src/test/ui/consts/const-eval/index-out-of-bounds-never-type.rs

```diff
@@ -9,11 +9,12 @@ struct PrintName<T>(T);
 impl<T> PrintName<T> {
     const VOID: ! = { let x = 0 * std::mem::size_of::<T>(); [][x] };
     //~^ WARN any use of this value will cause an error
+    //~| ERROR this operation will panic at runtime [unconditional_panic]
```

Comment:
please add `#![warn(unconditional_panic)]` to this file so we test the original thing again. This line can the be changed to
 
```suggestion
    //~| WARN this operation will panic at runtime [unconditional_panic]
```